### PR TITLE
[FEATURE] SAM - Zanshin Support

### PR DESCRIPTION
**Pull request by mikel-gh**
Replace Shinten with Zanshin while under the effect of Zanshin Ready.

Confirmed the action ID for Zanshin using ReAction.

image

### CODE REVIEW

File: XIVComboExpanded/Combos/SAM.cs [hunk 1]
```diff
@@ -296,6 +297,12 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                         return SAM.HissatsuGuren;
                 }
             }
+
+            if (IsEnabled(CustomComboPreset.SamuraiShintenZanshinFeature))
+            {
+                if (level >= SAM.Levels.Zanshin && HasEffect(SAM.Buffs.ZanshinReady))
```
Comment (mikel-gh) [inline XIVComboExpanded/Combos/SAM.cs:303]: SAM.Levels and SAM.Buffs were already updated, so only the action ID `SAM.Zanshin` needed to be added.

**Comment by MKhayle** [comment]
Oh, I guess that's because Ikishoten already has combos on it, forgot about that. This is indeed a viable option

**Comment by MKhayle** [comment]
if you can move it on top of the combo (from line 301 to 283) I'll merge that PR

**Comment by MKhayle** [comment]
nevermind, I just realized I could edit that PR
merging it now

**Pull request merged**